### ISSUE TITLE
Accuracy reported is different from accuracy through reproduced models

### ISSUE THREAD

**Issue by eyeshoe** [qfgaohao/pytorch-ssd#46]
I notice that when I run the vgg and mobilenet v2 models that you have already trained I get very similar accuracy to what you've reported, but when I try to reproduce those models through the method you suggest, I get significantly lower (20%) accuracy. Do you have any suggestions about how to improve that?

**Comment by pannaf** [comment]
Same. For mobilenet v2, with the default suggestion, I get something like:

Average Precision Per-class:
aeroplane: 0.5657933307950505
bicycle: 0.5958365508753679
bird: 0.4086400403629856
boat: 0.39042670180981476
bottle: 0.2186384020784879
bus: 0.6301307106013779
car: 0.6580358771149338
cat: 0.6643675101095109
chair: 0.3050131169713594
cow: 0.3497012449605594
diningtable: 0.5554262138275705
dog: 0.5491393700338869
horse: 0.6335805483851406
motorbike: 0.6147627261899834
person: 0.5939436865972345
pottedplant: 0.2141154264353295
sheep: 0.4522329331115968
sofa: 0.5716547884678509
train: 0.7024842940363902
tvmonitor: 0.5006772289276247

Average Precision Across All Classes:0.5087300350846028

**Comment by qfgaohao** [comment]
@eyeshoe @pannaf Have you used the pretrained mobilenetv2 models?

**Comment by pannaf** [comment]
No, I have not. I may have misunderstood, but I thought OP was asking about how to get the same accuracy with re-training from the ImageNet weights.

**Comment by eyeshoe** [comment]
I have used the pretrained models, but I'm curious how you actually trained them- I was confused because I thought that the process you explain to train is actually how you trained those models.

**Comment by wangnamu** [comment]
就要有回报。 yn

A这次 vv v，您这；个。。zGNZ

发自我的 iPhone

**Comment by STARK1234** [comment]
I used the pretrained model, and I set num_epochs 200. Only reach 0.66

**Comment by pannaf** [comment]
Have you tried running it again? Could have just been an unlucky initialization.

**Comment by STARK1234** [comment]
I will give a try. Have you reached the reported performance? How many epochs?

**Comment by pannaf** [comment]
Yes, by 200 epochs I reached the reported performance.

**Comment by STARK1234** [comment]
Okay, thx.
